### ISSUE TITLE
Improve description of the `filename` parameter of `loadwallet` RPC

### ISSUE THREAD

**Issue by murchandamus** [bitcoin/bitcoin#30269]
### Motivation

As the documentation for the [`loadwallet`](https://bitcoincore.org/en/doc/27.0.0/rpc/wallet/loadwallet/) RPC describes, the syntax is:

    loadwallet "filename" ( load_on_startup )

And it further specifies:

    Arguments:
    1. filename (string, required) The wallet directory or .dat file.
    …

What’s not clearly specified in the documentation is that you are to provide the filename _relative to the wallet directory_ `~/.bitcoin/regtest/wallets`. It would be great if the documentation were improved in that regard.

### Possible solution

- Improve the documentation for `loadwallet` to specify that the filename should be provided as the relative path from the wallet directory.
- Add an example where a wallet is loaded from a different directory

### Useful Skills

* `git` fundamentals

### Guidance for new contributors

Want to work on this issue?

For guidance on contributing, please read [CONTRIBUTING.md](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md) before opening your pull request.

**Comment by MukulKolpe** [comment]
Hey @murchandamus, can I work on this issue?

**Comment by willcl-ark** [comment]
@MukulKolpe absolutely. We don't generally assign issues in this project. Anyone is free to open a PR fixing any issue.

As @murchandamus advised in OP, please take care to:

> ...  read [CONTRIBUTING.md](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md?rgh-link-date=2024-06-11T17%3A09%3A29Z) before opening your pull request.

If you have any questions about the issue at hand here, or generally opening a Pull Request in this repo, feel free to ask away in here :)

**Comment by murchandamus** [comment]
Sure! For context, the following Bitcoin Stack Exchange topic was what inspired me to open the issue: https://bitcoin.stackexchange.com/q/123331/5406

**Comment by Akashkuvar** [comment]
loadwallet "filename" ( load_on_startup )

**Comment by Akashkuvar** [comment]
loadwallet "filename" ( load_on_startup )

**Comment by Akashkuvar** [comment]
What price

**Comment by am-sq** [comment]
Wanted to explicitly mention I have opened https://github.com/bitcoin/bitcoin/pull/30302 to resolve this issue. Thanks in advance for guidance and review.

**Comment by chennleyi** [comment]
Hi @murchandamus, is this issue about to get resolved? can I work on this issue? I am new to this.

**Comment by paulmaleauxney** [comment]
@murchandamus if this is still open I can work in it. Please let me know.

**Comment by krishpranav** [comment]
Hi, is it still open? i am looking forward to contribute.

**Comment by fanquake** [comment]
Closing, given #30302 has been merged.
